lib/broadway/processor.ex

```diff
@@ -72,6 +77,15 @@ defmodule Broadway.Processor do
         )
     end
 
+    metadata = %{
+      name: state.name,
+      successful_messages_to_ack: successful_messages_to_ack,
+      successful_messages_to_forward: successful_messages_to_forward,
+      failed_messages: failed_messages
+    }
+
+    measurements = %{time: System.monotonic_time()}
+    :telemetry.execute([:broadway, :processor, :stop], measurements, metadata)
```

Comment:
If I follow correctly, wouldn't you want to capture the duration of the processor event here? ie `System.monotonic_time() - start_time` since on the event handling side of telemetry you wouldn't know what event you are performing the duration calculation on? (if that makes sense)